certification/pyxis/pyxis.go

```diff
@@ -126,6 +127,59 @@ func (p *pyxisClient) getImage(ctx context.Context, dockerImageDigest string) (*
 	return &data.Data[0], nil
 }
 
+// FindImagesByDigest uses an unauthenticated call to find_images() graphql function, and will
+// return a slice of CertImages. It accepts a slice of image digests. Additional fields
+// for return should be specified in ContainerImage
+func (p *pyxisClient) FindImagesByDigest(ctx context.Context, digests []string) ([]CertImage, error) {
+	// our graphQL query
```

Comment:
Should we bail early if `digests` is empty?

Reply:
That's a great suggestion.